docs/plugin-vidarr.md

```diff
@@ -22,5 +23,7 @@ window is exceeded, they will be run as dry-run.
 
 The Vidarr plugin provides an input format: `vidarr_analysis` which provides the file type analyis provenance data.
 
+The `"analysisTypes"` specifies the type of analysis desired. The default is only files.
```

Comment:
Probably worth linking to the [Vidarr doc for analysis type](https://github.com/oicr-gsi/vidarr/blob/757cdb311109aefa6bd650172bd78fed04507fb7/vidarr-pluginapi/src/main/java/ca/on/oicr/gsi/vidarr/api/AnalysisOutputType.java) since it's controlled vocabulary. 

Also, it doesn't look like "FILES" is set as a default anywhere in the code; only in your example, so please remove or rework the second sentence.